src/server/test/web/readingsCompareGroupFlow.js

```diff
@@ -7,19 +7,116 @@
 	See: https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md for information.
 */
 const { chai, mocha, app } = require('../common');
+const Unit = require('../../models/Unit');
 const { prepareTest,
 	expectCompareToEqualExpected,
 	getUnitId,
+	GROUP_ID,
 	METER_ID,
-	unitDatakWh,
-	conversionDatakWh,
-	meterDatakWh } = require('../../util/readingsUtils');
+	unitDatakWh, 
+	conversionDatakWh
+ } = require('../../util/readingsUtils');
 
 mocha.describe('readings API', () => {
 	mocha.describe('readings test, test if data returned by API is as expected', () => {
 		mocha.describe('for compare charts', () => {
 			mocha.describe('for groups', () => {
-				// Add CG15 here
+				// Test CG15					
+				mocha.it(' 7 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and flow units & kW as kW ', async () => {
```

Comment:
Could you please add CG15: at the start of the description?

Reply:
I have added CG15 to the test description